src/databricks/labs/ucx/config.py

```diff
@@ -41,6 +41,9 @@ class WorkspaceConfig:  # pylint: disable=too-many-instance-attributes
     # Flag to see if terraform has been used for deploying certain entities
     is_terraform_used: bool = False
 
+    # Whether the assessment should capture a specific list of databases, if not specified, it will list all databases.
+    include_databases: list[str] | None = None
```

Comment:
can you document it in the readme?
https://github.com/databrickslabs/ucx?tab=readme-ov-file#open-remote-config-command